x-pack/plugins/event_log/server/es/documents.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EsNames } from './names';
+import mappings from '../../generated/mappings.json';
+
+export function getIndexTemplate(esNames: EsNames, ilmExists: boolean) {
+  const indexTemplateBody: any = {
+    index_patterns: [esNames.indexPattern],
+    aliases: {
+      [esNames.alias]: {},
+    },
+    settings: {
+      number_of_shards: 1,
+      number_of_replicas: 1,
+      'index.lifecycle.name': esNames.ilmPolicy,
+      'index.lifecycle.rollover_alias': esNames.alias,
+    },
+    mappings,
+  };
+
+  if (!ilmExists) {
+    delete indexTemplateBody.settings['index.lifecycle.name'];
+    delete indexTemplateBody.settings['index.lifecycle.rollover_alias'];
+  }
+
+  return indexTemplateBody;
+}
+
+export function getIlmPolicy() {
```

Comment:
Same question with these variables.

Reply:
This one is somewhat interesting.  I we have to have SOME KIND of ILM policy to get everything set up, but it's a one-time thing.  Once it's been created, we'll never touch it again.  The customer is expected to customize if they want.  So hardly seems worthy of configurability.

Reply:
That's true, users would have to go elsewhere to change it after initial setup.